internal/stringutil/addr.go

```diff
@@ -60,8 +60,13 @@ func EnsureCommaDelimitedAddresses(s string) string {
 			}
 			if inDomain {
 				if r == ';' {
-					sb.WriteRune(r)
-					break
+					inDomain = false
+					if i == len(s)-1 { // if last one
```

Comment:
```suggestion
					if i == len(s)-1 {
						// Preserve trailing semicolon.
```

Reply:
It won't preserve trailing semicolon, it will omit it.
I've updated comment